.config/config.cfg

```diff
@@ -8,6 +8,9 @@
     ALLOWED_HOSTS=127.0.0.1
     SECRET_KEY=<secret_key>
     DJANGO_DB=DEFAULT
+    WAM_EXCHANGE_ACCOUNT=<Microsoft Exchange Account, e.g. account_name@domain.tld>
+    WAM_EXCHANGE_EMAIL=<Microsoft Exchange E-Mail Adress, e.g. account_mail@domain.tld>
+    WAM_EXCHANGE_PW=<Microsoft Exchange Account Password>

```

Comment:
Writing a password in an uncrypted file???

Reply:
I do not have a better idea right now as stated in #65. Hashes do not make sense here.
Do you?

As a first step, we should change config file permissions on the server to 600 (-rw-------). Agree, @henhuy?